src/plugins/core/conditional_format.ts

```diff
@@ -57,6 +70,203 @@ export class ConditionalFormatPlugin
 
   readonly cfRules: { [sheet: string]: ConditionalFormatInternal[] } = {};
 
+  constructor(config: CorePluginConfig) {
+    super(config);
+    clipboardHandlersRegistries.cellHandlers.add("CONDITIONAL_FORMAT", {
+      copy: (
```

Comment:
I'm actually wondering it this needs to/should be defined here in the (core) plugins themselves. It could also be defined elsewhere (as long as it's extensible for each plugin).
Something that looks weird is they are given the UI getters and the UI dispatch